src/evict/evict_lru.c

```diff
@@ -914,6 +917,17 @@ __wt_evict_file_exclusive_on(WT_SESSION_IMPL *session)
     if (F_ISSET(session, WT_SESSION_PREFETCH_ENABLED))
         WT_ERR(__wt_conn_prefetch_clear_tree(session, false));
 
+    /*
```

Comment:
This code is part of the prefetch, I feel it is better to be inside the  `__wt_conn_prefetch_clear_tree` function.

Reply:
Thanks! [ae383e8](https://github.com/wiredtiger/wiredtiger/pull/10189/commits/ae383e82604d374efa961601fb9e9406cf492771)